stonesoup/dataassociator/mfa.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+
+from .base import DataAssociator
+from ..base import Property
+from ..hypothesiser import Hypothesiser
+from ..types.multihypothesis import MultipleHypothesis
+from .mfa_init import init_hyp_info, Hyp
+from .mfa_step import MAX_ITERATION_COUNT, AlgorithmState, algorithm_step, prune_hypotheses
+
+
+class MFADataAssociator(DataAssociator):
+    """Data associator using multi-frame assignment algorithm over a sliding window."""
```

Comment:
Capitalising the words in a spelt out initialism is not standard English (unless the term is a proper noun anyway e.g. UN) according to all style guides I've looked at on the subject (e.g. Chicago, The Economist). But it does seem to be a curiously common habit in the defence industry and in any case I'm still happy to make the change if you wish.

Reply:
Ok fair enough.  I just thought it might add clarity so someone can quickly see what "MFA" actually stands for, but it's not a big deal so I'll leave it up to you.

Reply:
Good point. I'll put the abbreviation in the comment too: "Data associator using multi-frame assignment (MFA) algorithm over a sliding window"